src/sharepoint/account/sharepoint-account.controller.js

```diff
@@ -182,8 +180,8 @@ angular
             this.accountIds = null;
 
             return this.sharepointService.getAccounts(this.exchangeId, this.search.value)
-                .then((accountIds) => {
-                    this.accountIds = accountIds;
+                .then((ids) => {
+                    this.accountIds = ids.map((accountId) => ({ accountId }));
```

Comment:
Please use `_.map(ids, (accountId) => (accountId) => ({ accountId }))`, we have chosen to use Lodash everywhere instead of native ES6 functions :smiley:

Reply:
Map is actually es5 and supported by IE9, so there's no reason to avoid it.